### PR TITLE
Adding TerminusDB

### PR DESCRIPTION
**Pull request by Cheukting**
Also arranging DBaaS section according to aphabetical order.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -620,12 +620,13 @@ Table of Contents
...
+   * [InfluxDB](https://www.influxdata.com/) — Timeseries database, free up to 3MB/5 minutes writes, 30MB/5 minutes reads and 10,000 cardinalities series
    * [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) — free tier gives 512 MB
    * [redislabs.com](https://redislabs.com/redis-cloud) — Redis as a Service, 30 MB and 30 concurrent connections free
    * [redsmin.com](https://www.redsmin.com/) — Online real-time monitoring and administration service for Redis, 1 Redis instance free
    * [scalingo.com](https://scalingo.com/) — Primarily a PaaS but offers a 128MB to 192MB free tier of MySQL, PostgreSQL or MongoDB
    * [skyvia.com](https://skyvia.com/) — Cloud Data Platform, offers free tier and all plans are completely free while in beta
-   * [InfluxDB](https://www.influxdata.com/) — Timeseries database, free up to 3MB/5 minutes writes, 30MB/5 minutes reads and 10,000 cardinalities series
+   * [TerminusDB](https://terminusdb.com/) — Open-source Graph database that store data like Git.
```
Comment (ripienaar) [inline README.md]: doesnt appear to be offered as a service with a free tier?

Comment (Cheukting) [reply README.md]: The hub service will come at the end of January and there will be a free tier. Right now it's a open-source database. Maybe another review after it is released?

Comment (ripienaar) [reply README.md]: sure, feel free to open a new PR then

Comment (sdil) [reply README.md]: image
Sorry but the free service is stated here

Comment (ripienaar) [reply README.md]: Comment was about terminusdb :)

**Pull request closed without merging**